### PR TITLE
Refactor project page-change styling-add ui componenets

### PR DESCRIPTION
**Pull request by MuhammetCoskun**
-restructured project page
-added new ui components

### CODE REVIEW

File: src/components/UI/ResponsiveTypography.tsx [hunk 1]
```diff
@@ -0,0 +1,30 @@
+import * as React from "react";
+import { Variant } from "@mui/material/styles/createTypography";
+import {
+  createTheme,
+  responsiveFontSizes,
+  ThemeProvider,
+} from "@mui/material/styles";
+import Typography from "@mui/material/Typography";
+
+let theme = createTheme({
```
Comment (wickY26) [inline src/components/UI/ResponsiveTypography.tsx:10]: Does it make sense to manage themes at different places?
This kind of specific styling using CSS would be much better.
Please try to use and manage your theme on the upper level and preferably from one component (like your Layout.tsx) so

File: src/components/Layout/Seo/index.tsx [hunk 2]
```diff
@@ -1,18 +1,15 @@
 import React from "react";
 import Head from "next/head";
 import config from "../../config";
-import { useRouter } from "next/router";
+import usePageTitle from "../../../hooks/usePageHeader";
```
Comment (wickY26) [inline src/components/Layout/Seo/index.tsx:4]: using consistent naming is a good practice. Please 'usePageHeader' as your import.

Comment (MuhammetCoskun) [reply src/components/Layout/Seo/index.tsx:4]: usePageHeader -> usePageTitle 
file name will be changed

File: src/components/UI/ResponsiveTypography.tsx (hunk 1 above)
Comment (MuhammetCoskun) [reply src/components/UI/ResponsiveTypography.tsx]: theme moved to layout 👍

**Pull request merged**